chapters/cli/exit.rst

```diff
@@ -85,3 +84,8 @@ For Flow, when you see the following output from the console, then it is shut do
     chunk_idx-6@6381[I]:msg_sent: 653 bytes_sent: 589 KB msg_recv: 326 bytes_recv:953 KB
     chunk_idx-6@6381[S]:terminated
     Flow@6331[S]:flow is closed and all resources should be released already, current build level is EMPTY
+    
+Using Jina remotely 
+^^^^^^^^^^^^^^    
```

Comment:
please make this `^^^` longer to cover the title

Reply:
oh, right, I will fix that too. Thanks for the info!

Reply:
ah ... it was hidden in the update. I will fix it now :)